### PR TITLE
Add 'Hagen Grote GmbH'

### PR DESCRIPTION
**Pull request by moriarty-jim**
Added mail order business/online shop "Hagen Grote GmbH".
Tested GDPR request successfully about 2 years ago.

### CODE REVIEW

File: companies/hagengrote.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Hagen Grote GmbH",
+    "address": "Grüner Dyk 36\n47803 Krefeld\nDeutschland",
+    "phone": "+49 2151 607090",
+    "fax": "+49 2151 6070999",
+    "email": "datenschutz@hagengrote.de",
```
Comment (baltpeter) [inline companies/hagengrote.json:13]: As this email is specific for privacy-related requests, we can set the `suggested-transport-medium` to `email`.

File: companies/hagengrote.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "slug": "hagengrote",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Hagen Grote GmbH",
+    "address": "Grüner Dyk 36\n47803 Krefeld\nDeutschland",
```
Comment (baltpeter) [inline companies/hagengrote.json:10]: If the privacy policy lists the address of an external data protection officer, we use that.

```suggestion
    "address": "c/o ZB Datenschutz und -sicherheit GmbH & Co. KG\nHansestraße A1/79\n51149 Köln\nDeutschland",
```

File: companies/hagengrote.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    "phone": "+49 2151 607090",
```
Comment (baltpeter) [inline companies/hagengrote.json:11]: Same for the phone number…

```suggestion
    "phone": "+49 2203 8984444",
```

File: companies/hagengrote.json (hunk 1 above)
Comment (baltpeter) [inline companies/hagengrote.json:13]: …and email. :D 

```suggestion
    "email": "dsb.hagengrote@zb-datenschutz.com",
```

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record! I have requested some changes below.

**Comment by moriarty-jim** [comment]
Thanks for your feedback! Sorry for the extra work. I'll know better next time.

**Review by baltpeter — APPROVED** [review]
Thank you!

**Comment by baltpeter** [comment]
> Thanks for your feedback! Sorry for the extra work. I'll know better next time.

No worries, that's what the review is there for, thanks for contributing!

**Pull request merged**